lib/appliance_console/service_group.rb

```diff
@@ -41,32 +28,36 @@ def restart
     end
 
     def enable
-      LinuxAdmin.run("chkconfig", :params => {"--add" => "miqtop"})  # Is this really needed?
-      service_command("enable")
+      enable_miqtop
+      SERVICES.each { |s| run_service(s, "enable") }
+      run_service(POSTGRESQL_SERVICE, "enable") if postgresql?
     end
 
     def disable
-      service_command("disable")
+      run_service(POSTGRESQL_SERVICE, "disable") unless postgresql?
     end
 
     def start
-      start_command
+      SERVICES.each { |s| run_detached_service(s, "start") }
```

Comment:
@kbrock I like this until we move some of this logic into LInuxAdmin.  It would be nice to not have to tell the service to run detached via method.  It sounds like we should have a regular service and a detached service that knows how to start itself.

Either way, this is a great change, I wasn't a fan of the initial indirection in the first place.  :smile: